docs/examples/crud-example.md

```diff
@@ -1,7 +1,231 @@
 ---
 id: crud-example
-title: CRUD Example
-sidebar_label: CRUD Example
+title: CRUD Examples
+sidebar_label: CRUD Examples
 ---
 
-Jules, your examples will go in this section!
+# `defineFetch` HTTP services
+
+Quick examples of GET, PUT, POST and DELETE `defineFetches`.
+
+## Example: GET with and without unique identifier
+
+```ts
+// component/actions/peopleFetch.ts
+
+import defineFetch from 'resift/defineFetch';
+
+export default defineFetch({
+  displayName: 'Get People',
+  make: () => ({
+    key: [],
+    request: () => ({ http }) =>
+      return http({
+        method: 'GET',
+        route: `/people`,
+      }),
+  }),
+});
+```
+
+```ts
+// component/actions/personFetch.ts
+
+import defineFetch from 'resift/defineFetch';
+
+export default defineFetch({
+  displayName: 'Get Person',
+  make: (personId: string) => ({
+    key: [personId],
+    request: () => ({ http }) =>
+      return http({
+        method: 'GET',
+        route: `/person/${personId}`,
+      }),
+  }),
+});
+```
+
+```tsx
+// Example PeopleFetcher component using `definefetch` above
+
+import React, { useEffect } from 'react';
+import useFetch from 'resift/useFetch';
+import useDispatch from 'resift/useDispatch';
+
+// Returned Component
+import PeopleList from './PeopleList';
+
+// Action in example above
+import makePeopleFetch from './actions/peopleFetch';
+import makePersonFetch from './actions/personFetch';
+
+function PeopleFetcher() {
+  const dispatch = useDispatch();
+
+  const peopleFetch = makePeopleFetch();
+  const personFetch = makePersonFetch('John Smith');
+
+  // On ComponentDidMount
+  useEffect(() => {
+    dispatch(peopleFetch());
+    dispatch(makePersonFetch());
+  }, [dispatch, makePersonFetch, peopleFetch]);
+
+  const [people, peopleStatus] = useFetch(peopleFetch());
+  const [person, personStatus] = useFetch(makePersonFetch());
+
+  return <PeopleList people={people} person={person} />;
+}
+
+export default PeopleFetcher;
+```
+
+<br />
+<br />
+<br />
+
+## Example: Post
+
+```ts
+// component/actions/createPerson.ts
+
+import defineFetch from 'resift/defineFetch';
+
+export default defineFetch({
+  displayName: 'Create Person',
+  make: () => ({
+    key: [],
+    request: (personName: string) => ({ http }) =>
+      return http({
+        method: 'POST',
+        route: `/person`,
+        data: {
+          name: personName
+        }
+      }),
+  }),
+});
+```
+
+<br />
+<br />
+<br />
+
+## Example: Delete
+
+```ts
+// component/actions/deletePerson.ts
+
+import defineFetch from 'resift/defineFetch';
+
+export default defineFetch({
+  displayName: 'Delete Person',
+  make: () => ({
```

Comment:
Why don't you pass the id in the make here like you did for the `getPersonFetch` below? I feel like these are the same(regardless of CRUD) so we should try to come up with a standard.